webapp/app.py

```diff
@@ -25,15 +25,189 @@ def utility_processor():
+            "type": "charm",
+            "publisher": "Amazon Web Services",
+            "sections": [{"featured": True, "name": "server-and-cloud"}],
+            "summary": "Universal Command Line Interface for"
+            " Amazon Web Services",
+            "title": "aws-cli",
+        },
+        {
+            "developer_validation": "unproven",
+            "icon_url": "https://dashboard.snapcraft.io/site_media/appmedia"
+            "/2018/03/rsz_android_studio_icon.png",
+            "media": [
+                {
+                    "type": "icon",
+                    "url": "https://dashboard.snapcraft.io/site_media"
+                    "/appmedia/2018/03/rsz_android_studio_icon.png",
+                },
+                {
+                    "type": "screenshot",
+                    "url": "https://dashboard.snapcraft.io/site_media"
+                    "/appmedia/2018/03/android-studio.png",
+                },
+                {
+                    "type": "screenshot",
+                    "url": "https://dashboard.snapcraft.io/site_media"
+                    "/appmedia/2019/04/Screenshot_20190418_002828.png",
+                },
+            ],
+            "origin": "snapcrafters",
+            "package_name": "android-studio",
+            "publisher": "Snapcrafters",
+            "sections": [{"featured": True, "name": "development"}],
+            "summary": "The IDE for Android",
+            "title": "Android Studio",
+        },
+    ]
+
+    context = {"results": search_results}
+    return render_template("index.html", **context)
 
 
 @app.route('/<regex("' + config.details_regex + '"):entity_name>')
 def details(entity_name):
-    # TODO this will not be required when we have the type from the API
-    entity_type = request.args.get("type", "charm")
+    # TO DO - get entity info from API
 
     context = {
-        "entity_type": entity_type,
+        "entity-id": "RT9mcUhVsRYrDLG8qnvGiy26NKvv6Qkd",
+        "entity_title": "VLC",
+        "package_name": "vlc",
+        "categories": [{"slug": "photo-and-video", "name": "Photo and Video"}],
+        "icon_url": "https://dashboard.snapcraft.io/site_media/appmedia/2016/"
+        "07/vlc.png",
+        "version": "3.0.8",
+        "license": "GPL-2.0+",
+        "publisher": "VideoLAN",
+        "username": "videolan",
+        "screenshots": [
+            {
+                "height": "None",
+                "type": "screenshot",
+                "url": "https://dashboard.snapcraft.io/site_media/appmedia/"
+                "2016/07/vlc-2.0-poney.jpg",
+                "width": "None",
+            },
+            {
+                "height": "None",
+                "type": "screenshot",
+                "url": "https://dashboard.snapcraft.io/site_media/appmedia/"
+                "2018/04/2018-04-26-161203_1126x899_scrot_yBFL55L.png",
+                "width": "None",
+            },
+        ],
+        "videos": [],
+        "publisher_snaps": "None",
```

Comment:
I *think* we should remove the following 3 lines.
`publisher_snaps`, `publisher_featured_snaps` and `has_publisher_page` as these are overrides we add on the frontend that won't be applicable to charmhub. This might change some of the checks you do in the templates!